test/unit/instance_method_test.rb

```diff
@@ -118,22 +117,12 @@ def self.method_x(&block)
     method.hide_original_method
     method.define_new_method
     method.remove_new_method
-    method.restore_original_method
```

Comment:
Since this test is named `test_should_restore_original_method_accepting_a_block_parameter ` and it seems to have been testing the implementation of `#restore_original_method` in the parent class, `StubbedMethod`, presumably we could delete the whole test...?

Reply:
👍